Notion Release Notes Agent/polling_server.py

```
@@ -0,0 +1,346 @@
+                    state = json.load(f)
+                    self.seen_prs = set(state.get("seen_prs", []))
+                    logger.info(f"📋 Loaded {len(self.seen_prs)} previously seen PRs")
+            except Exception as e:
+                logger.warning(f"⚠️ Could not load state file: {e}")
+
+    def _save_state(self):
+        """Save seen PR numbers to state file"""
+        try:
+            with open(self.state_file, "w") as f:
+                json.dump({"seen_prs": list(self.seen_prs)}, f)
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)